website/docs/r/aviatrix_account_user.html.markdown

```diff
@@ -29,6 +31,12 @@ The following arguments are supported for creating user account:
 * `email` - (Required) Email of address of account user to be created.
 * `password` - (Required) Login password for the account user to be created. If password is changed, current account will be destroyed and a new account will be created.
 
+The following arguments are deprecated:
+
+* `account_name` - (Required) Cloud account name of user to be created. Deprecated as of Aviatrix provider R2.13 (Controller 5.4) due to RBAC implementation.
+
+-> **NOTE:** `account_name` - If you are using/upgraded to Aviatrix Terraform Provider R2.13+, and an **aviatrix_account_user** resource was originally created with a provider version <R2.13, you must remove this attribute and perform a 'terraform refresh' to rectify the state file.
```

Comment:
You could mention here that the aviatrix_rbac_group_user_attachment resource can be used to attach to any group